pkg/gcptarget/library.go

```diff
@@ -0,0 +1,71 @@
+package gcptarget
+
+import "text/template"
+
+const libraryTemplateSrc = `package target
+
+matching_constraints[constraint] {
+	asset := input.review
+	constraint := {{.ConstraintsRoot}}[_][_]
+	spec := get_default(constraint, "spec", {})
+	match := get_default(spec, "match", {})
+
+	# Default matcher behavior is to match everything.
+	target := get_default(match, "target", ["organization/*"])
+	# TODO: Retire the "gcp" wrapper.
+	# See https://github.com/forseti-security/config-validator/issues/42
+	gcp := get_default(match, "gcp", {})
+	gcp_target := get_default(gcp, "target", target)
+	re_match(gcp_target[_], asset.ancestry_path)
```

Comment:
Are we meant to be using regex matching or prefix and/or glob matching?

Reply:
turns out it's using regex, but the docs indicate glob, it's going to be interesting.

Reply:
Rego also has Glob:

https://www.openpolicyagent.org/docs/latest/policy-reference/#glob

And the prefix() check is an option for simplicity.

I don't think it's my place to say any one of these is better than the other, however.